www/index.html

```diff
@@ -385,9 +385,9 @@ <h3 id="feedback" data-i18n="about-feedback">Feedback / helping / contributing</
                     <h3 id="credits" data-i18n="about-credits">Credits</h3>
                     <p data-i18n="about-credits-gratitude">We owe a big debt of gratitude to:
```

Comment:
Although your code works, I think it only works because the browser auto-adds a close `</p>` at the end of this line. So in effect, I believe it works due to a side effect! We probably shouldn't rely on that.

All you have to do is add a `</p>` at the end of this line, and remove the `z/p>` at the bottom of the block (see below).

Reply:
Ahhh I see. I'll fix it.